tests/python/relay/test_pass_convert_op_layout.py

```diff
@@ -625,3 +685,5 @@ def expected():
     test_qnn_conv_requantize_convert_layout()
     test_qnn_conv_concat_convert_layout()
     test_qnn_conv_add_convert_layout()
+    test_conv_convert_kernel_layout()
+    test_default_keyword()
```

Comment:
A soft suggestion - Does it make sense to add a test that tests conversion for 2 operators simultaneously - Like conv3d and conv2d in the same network? Or qnn.conv2d and nn.conv2d in the same network.

Reply:
I think that's good to check, will add that.